test/over_react/util/react_wrappers_test.dart

```diff
@@ -226,51 +226,45 @@ main() {
         });
       });
 
-      group('updates the "key" and "ref" props properly', () {
-        const Map originalKeyRefProps = const {
+      group('updates the "key" props properly', () {
+        const Map originalKeyProps = const {
           'key': 'original',
-          'ref': 'original'
         };
 
-        const Map overrideKeyRefProps = const {
+        const Map overrideKeyProps = const {
           'key': 'clone',
-          'ref': 'clone'
         };
 
         test('for a plain React JS component', () {
-          var original = (Dom.div()..addProps(originalKeyRefProps))(testChildren);
-          var clone = cloneElement(original, overrideKeyRefProps);
+          var original = (Dom.div()..addProps(originalKeyProps))(testChildren);
+          var clone = cloneElement(original, overrideKeyProps);
 
-          // Verify that "key" and "ref" are overridden according to React
-          expect(clone.key, equals(overrideKeyRefProps['key']));
-          expect(clone.ref, equals(overrideKeyRefProps['ref']));
```

Comment:
If you end up calling this with a callback ref, then watch out: this assertion will fail because the ref on the clone will be the wrapper react-dart adds. It'd probably be easiest to test functionally by rendering the component and making sure the right function is called.